my-page-app/src/components/ui/monthCalendar.tsx

```diff
@@ -43,39 +43,37 @@ function MonthCalendar({
     getYourBookings()
   }, [])
 
-  const fetchBookings = async () => {
-    try {
-      const currentDate = new Date()
-      const unformattedStartDate = new Date(
-        currentDate.getFullYear(),
-        currentDate.getMonth() - 6,
-        currentDate.getDate(),
-      )
-      const unformattedEndDate = new Date(
-        currentDate.getFullYear(),
-        currentDate.getMonth() + 12,
-        currentDate.getDate(),
-      )
-      const startDate = format(unformattedStartDate, 'yyyy-MM-dd')
-      const endDate = format(unformattedEndDate, 'yyyy-MM-dd')
-      //Todo: change the start and enddates later once booking is in place so it is more than just a month but six months back and twelve months forward. These control the time period in which bookings will be rendered on the calendar.
-
-      const bookings = await ApiService.getBookings(startDate, endDate)
-      setBookings(bookings)
-    } catch (error) {
-      console.error('Error:', error)
-    }
-  }
-
-  useEffect(() => {
-    fetchBookings()
-  }, [])
+    const { data: bookings, refetch: refetchBookings } = useQuery<Booking[]>(
+        "bookings",
+        async () => {
+            const currentDate = new Date()
+            const unformattedStartDate = new Date(
+                currentDate.getFullYear(),
+                currentDate.getMonth() - 2,
+                currentDate.getDate(),
+            )
+            const unformattedEndDate = new Date(
+                currentDate.getFullYear(),
+                currentDate.getMonth() + 1,
+                currentDate.getDate(),
+            )
```

Comment:
Se om dette fungerer:

const startDate =
```suggestion
            // in imports:
            import { add, sub } from 'date-fns'
            //her:
            const startDate = format(sub(new Date(), { month: 2 })), 'yyyy-MM-dd')
            const endDate = format(add(new Date(), { month: 1 })), 'yyyy-MM-dd')
```